packages/blanket/src/Blanket.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { withTheme } from 'styled-components';
+
+import { Root } from './styled';
+import { IBlanketProps } from './interfaces';
+
+
+export const PureBlanket = ({
+  appearance,
+  baseAppearance,
+  canClickThrough = false,
+  isTinted = false,
+  onBlanketClicked = () => {},
+}: IBlanketProps) => {
+  const onClick = canClickThrough ? undefined : onBlanketClicked;
+
+  return (
+    <Root
+      appearance={ appearance }
+      baseAppearance={ baseAppearance }
+      canClickThrough={ canClickThrough }
+      isTinted={ isTinted }
+      onClick={ onClick }
+    />
+  );
+};
+
+export const Blanket = React.memo(withTheme(PureBlanket));
```

Comment:
withTheme is unnecessary because Root component has access to theme by default.

Reply:
![image](https://user-images.githubusercontent.com/8888560/70624714-e63db380-1c31-11ea-8e91-b5f7c482f73a.png)
No, it is not work without 'withTheme'